src/Sidekick.Business/Apis/Poe/Parser/ParserService.cs

```diff
@@ -0,0 +1,318 @@
+            {
+                { Rarity.Normal, new Regex(Regex.Escape(languageProvider.Language.RarityNormal)) },
+                { Rarity.Magic, new Regex(Regex.Escape(languageProvider.Language.RarityMagic)) },
+                { Rarity.Rare, new Regex(Regex.Escape(languageProvider.Language.RarityRare)) },
+                { Rarity.Unique, new Regex(Regex.Escape(languageProvider.Language.RarityUnique)) },
+                { Rarity.Currency, new Regex(Regex.Escape(languageProvider.Language.RarityCurrency)) },
+                { Rarity.Gem, new Regex(Regex.Escape(languageProvider.Language.RarityGem)) },
+                { Rarity.DivinationCard, new Regex(Regex.Escape(languageProvider.Language.RarityDivinationCard)) }
+            };
+
+            ItemLevelPattern = new Regex($"[\\r\\n]{Regex.Escape(languageProvider.Language.DescriptionItemLevel)}[^\\r\\n\\d]*(\\d+)");
+            UnidentifiedPattern = new Regex($"[\\r\\n]{Regex.Escape(languageProvider.Language.DescriptionUnidentified)}");
+            CorruptedPattern = new Regex($"[\\r\\n]{Regex.Escape(languageProvider.Language.DescriptionCorrupted)}");
+        }
+
+        private void ParseHeader(ref ParsedItem item, ref string input)
+        {
+            var lines = NewlinePattern.Split(input);
+            var blocks = SeparatorPattern.Split(input);
+
+            item.Rarity = GetRarity(lines[0]);
+
+            var dataItem = itemDataService.GetItem(blocks[0]);
+
+            item.Name = dataItem.Name;
+            item.TypeLine = dataItem.Type;
+
+            item.ItemLevel = GetInt(ItemLevelPattern, input);
+            item.Identified = !UnidentifiedPattern.IsMatch(input);
+            item.Corrupted = CorruptedPattern.IsMatch(input);
+        }
+
+        private Rarity GetRarity(string rarityString)
+        {
+            foreach (var pattern in RarityPatterns)
+            {
+                if (pattern.Value.IsMatch(rarityString))
+                {
+                    return pattern.Key;
+                }
+            }
+            throw new Exception("Can't parse rarity.");
+        }
+        #endregion
+
+        #region Properties (Armour, Evasion, Energy Shield, Quality, Level)
+        private Regex ArmorPattern { get; set; }
+        private Regex EnergyShieldPattern { get; set; }
+        private Regex EvasionPattern { get; set; }
+        private Regex QualityPattern { get; set; }
+        private Regex LevelPattern { get; set; }
+        private Regex MapTierPattern { get; set; }
+        private Regex ItemQuantityPattern { get; set; }
+        private Regex ItemRarityPattern { get; set; }
+        private Regex MonsterPackSizePattern { get; set; }
+        private Regex AttacksPerSecondPattern { get; set; }
+        private Regex CriticalStrikeChancePattern { get; set; }
+        private Regex ElementalDamagePattern { get; set; }
+        private Regex PhysicalDamagePattern { get; set; }
+        private Regex BlightedPattern { get; set; }
```

Comment:
These groups of regexes really should live in their own classes, I think. So you'd for example have a PropertyPatterns class (or something like that) that contains these properties and handles initalizing the patterns. It would help clean up this class immensely and improve readability.